### PR TITLE
Release/1.6.0 0.25b0

### PR DESCRIPTION
**Pull request by lzchen**

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/__init__.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def _instrument(self, **kwargs):
 
     # pylint:disable=no-self-use
     def _uninstrument(self, **kwargs):
-        """"Disable aiopg instrumentation"""
+        """ "Disable aiopg instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/__init__.py:94]: Typo?

File: instrumentation/opentelemetry-instrumentation-mysql/src/opentelemetry/instrumentation/mysql/__init__.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def _instrument(self, **kwargs):
         )
 
     def _uninstrument(self, **kwargs):
-        """"Disable MySQL instrumentation"""
+        """ "Disable MySQL instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-mysql/src/opentelemetry/instrumentation/mysql/__init__.py:81]: Typo?

File: instrumentation/opentelemetry-instrumentation-psycopg2/src/opentelemetry/instrumentation/psycopg2/__init__.py [hunk 3]
```diff
@@ -89,7 +89,7 @@ def _instrument(self, **kwargs):
         )
 
     def _uninstrument(self, **kwargs):
-        """"Disable Psycopg2 instrumentation"""
+        """ "Disable Psycopg2 instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-psycopg2/src/opentelemetry/instrumentation/psycopg2/__init__.py:92]: Typo?

File: instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py [hunk 4]
```diff
@@ -37,7 +37,7 @@
 class PymemcacheClientTestCase(
     TestBase
 ):  # pylint: disable=too-many-public-methods
-    """ Tests for a patched pymemcache.client.base.Client. """
+    """Tests for a patched pymemcache.client.base.Client."""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py:40]: This is a good change, but I think we should remove the `.` at the end so it doesn't look like we forgot to add something else to the import path?
```suggestion
    """Tests for a patched pymemcache.client.base.Client"""
```

File: instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py [hunk 5]
```diff
@@ -470,7 +470,7 @@ def test_uninstrumented(self):
 
 
 class PymemcacheHashClientTestCase(TestBase):
-    """ Tests for a patched pymemcache.client.hash.HashClient. """
+    """Tests for a patched pymemcache.client.hash.HashClient."""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py:473]: Maybe remove the `.` at the end?

```suggestion
    """Tests for a patched pymemcache.client.hash.HashClient"""
```

File: instrumentation/opentelemetry-instrumentation-pymysql/src/opentelemetry/instrumentation/pymysql/__init__.py [hunk 6]
```diff
@@ -79,7 +79,7 @@ def _instrument(self, **kwargs):
         )
 
     def _uninstrument(self, **kwargs):
-        """"Disable PyMySQL instrumentation"""
+        """ "Disable PyMySQL instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-pymysql/src/opentelemetry/instrumentation/pymysql/__init__.py:82]: Typo?

File: instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/__init__.py [hunk 7]
```diff
@@ -138,7 +138,7 @@ def _instrument(self, **kwargs):
         _wrap("pyramid.config", "Configurator.__init__", _traced_init)
 
     def _uninstrument(self, **kwargs):
-        """"Disable Pyramid instrumentation"""
+        """ "Disable Pyramid instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/__init__.py:141]: Typo?

File: instrumentation/opentelemetry-instrumentation-sqlite3/src/opentelemetry/instrumentation/sqlite3/__init__.py [hunk 8]
```diff
@@ -74,7 +74,7 @@ def _instrument(self, **kwargs):
         )
 
     def _uninstrument(self, **kwargs):
-        """"Disable SQLite3 instrumentation"""
+        """ "Disable SQLite3 instrumentation"""
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-sqlite3/src/opentelemetry/instrumentation/sqlite3/__init__.py:77]: Typo?

**Comment by lzchen** [comment]
@NathanielRN 
Hmm weird, I think running lint might've changed a lot of these docstrings. Let me revert them.

File: CHANGELOG.md [hunk 9]
```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.5.0-0.24b0...HEAD)
+## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.6.0-0.25b0...HEAD)
+
+## [1.6.0-0.25b0](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.6.0-0.25b0) - 2021-10-12
```
Comment (lonewolf3739) [inline CHANGELOG.md:10]: update the date?

File: propagator/opentelemetry-propagator-aws-xray/src/opentelemetry/propagators/aws/version.py [hunk 10]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.0.0"
+__version__ = "0.25b0"
```
Comment (lonewolf3739) [inline propagator/opentelemetry-propagator-aws-xray/src/opentelemetry/propagators/aws/version.py:15]: is this intended?

**Pull request merged**